haiku-apps/nanodot/nanodot-1.1b.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A minimal pattern-based MIDI sequencer"
+DESCRIPTION="nanodot is a rather minimal pattern-based MIDI \
+sequencer originally written for BeOS R3. Its main purpose in life was/is \
+to teach me (and hopefully others) the basics of writing MIDI apps for BeOS; \
+it makes for a nice drum-programming 'scratchpad', and is a generally entertaining \
+way to waste a few hours."
+HOMEPAGE="https://github.com/HaikuArchives/NanoDot"
+COPYRIGHT="1999 Eric Moon"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="80cbeedb3f86ddad8a92bfcf7212704f1528da0f"
+SOURCE_URI="https://github.com/HaikuArchives/NanoDot/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="6cd316390ec1939f7b8e21b1495ec9fffccc3d859b8d0caddb5ca6b6b56399cc"
+SOURCE_FILENAME="NanoDot-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="NanoDot-$srcGitRev"
+PATCHES="nanodot-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	nanodot = $portVersion
+	app:nanodot = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD_PREREQUIRES="
+	cmd:g++
+	"
+
+BUILD()
+{
+	cd src
+	g++ -c *.cpp
+	g++ -o nanodot *.o -lbe -lmidi ${LIBSTDCPP}
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cp -a src/nanodot $appsDir
```

Comment:
Oops, I forgot to add this suggestion:
```diff
 INSTALL()
 {
-	mkdir -p $appsDir
+	install -d $appsDir
-	cp -a src/nanodot $appsDir
+	install -t $appsDir src/nanodot
-	addAppDeskbarSymlink $appsDir/nanodot "NanoDot"
+	addAppDeskbarSymlink $appsDir/nanodot NanoDot
 }
```

Regarding the license (near the top of the recipe), if you don't know what it is you can put just:
```
LICENSE="unknown"
```

Reply:
Thanks for your advice. But `LICENSE="unknown"` cannot build on x86_64(invalid licence).